Makefile

```diff
@@ -19,7 +19,15 @@ GOLANGCI_LINT_VERSION ?= "v1.37.0"
 REDIS_VERSION ?= "5.0.8"
 TOOLS_MOD_DIR := ./internal/tools
 
-BUILD_TAGS:=brotli
+BUILD_TAGS:=
+
+# the original Makefile had 'brotli' always included in BUILD_TAGS
+# for this historical reason:
+#  - brotli is an *opt-out* feature (included by default, use NO_LIBBROTLI to *not* include it),
+#  - the other libs are opt-in features (not included by default, use USE_LIBSODIUM or USE_LZO to include them)
+ifndef NO_LIBBROTLI
```

Comment:
I propose to rename it like `USE_BROTLI`, to match the `USE_LZO` and `USE_LIBSODIUM` options

Reply:
we certainly can, but this is a breaking change.

meaning :
* within wal-g project : your test suite and your build pipeline would have to be changed too (I don't think you want to relase versions of wal-g that do not work for existing users, am I correct ?)
* for outside users of the project that compile your project : updating to latest version would silently change the result of their build -- if they intended to keep libbrotli, they too would have to adapt their build scripts / test suites.

Is that kind of change ok with you ?

Reply:
We can set it to `1` by default

Reply:
happy to oblige :wink: